docs/_ug/Commands.md

```diff
@@ -1,5 +1,6 @@
 <!-- markdownlint-disable-file first-line-h1 -->
-This section shares with you on how to use each command in detail.
+This section shares with you on how to use each [[command:command]] in detail.
```

Comment:
We can simplify this, since the key is the same as the alias:

```suggestion
This section shares with you on how to use each [[ command ]] in detail.
```